game-core/src/main/java/games/strategy/triplea/delegate/DiceRoll.java

```diff
@@ -424,37 +425,19 @@ private static void sortAaHighToLow(
     sortAaHighToLow(units, data, defending, new HashMap<>());
   }
 
-  private static void sortAaHighToLow(
+  @VisibleForTesting
```

Comment:
Instead of making this visible for testing, any thoughts to extract this method to a package-protected utility class, something like "DiceRollSorter"?

Reply:
Not feasible for this PR.
The method makes use of `getMaxAaAttackAndDiceSides`, which could be used in a static way, but at this point we'd create a utility class that uses the class it is called from